src/components/StorageOverview/Alerts/Alerts.js

```diff
@@ -1,39 +1,94 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Row, Col } from 'patternfly-react';
 
 import {
   DashboardCard,
   DashboardCardBody,
   DashboardCardHeader,
   DashboardCardTitle,
 } from '../../Dashboard/DashboardCard';
+import { DashboardCardActionsBody } from '../../Dashboard/DashboardCard/DashboardCardActionsBody';
+import { Dropdown } from '../../Form/Dropdown';
+
 import { StorageOverviewContext } from '../StorageOverviewContext';
 import { AlertsBody } from '../../Dashboard/Alert/AlertsBody';
 import { filterAlerts } from './utils';
 import { AlertItem } from '../../Dashboard/Alert/AlertItem';
+import { getAlertSeverity } from '../../../selectors/prometheus/alerts';
+import { ALL, WARNING, CRITICAL } from './strings';
+
+const alertsFilterOptions = [ALL, CRITICAL, WARNING];
 
-export const Alerts = ({ alertsResponse, className }) => {
-  if (!Array.isArray(alertsResponse)) {
-    return null;
+export class Alerts extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      type: alertsFilterOptions[0],
+    };
   }
 
-  const alerts = filterAlerts(alertsResponse);
-
-  return alerts.length > 0 ? (
-    <DashboardCard className={className}>
-      <DashboardCardHeader>
-        <DashboardCardTitle>Alerts</DashboardCardTitle>
-      </DashboardCardHeader>
-      <DashboardCardBody>
-        <AlertsBody>
-          {alerts.map((alert, index) => (
-            <AlertItem key={`alert-${index}`} alert={alert} />
-          ))}
-        </AlertsBody>
-      </DashboardCardBody>
-    </DashboardCard>
-  ) : null;
-};
+  onDropdownOptionChange = newVal => this.setState({ type: newVal }, this.getAlerts);
```

Comment:
what is the `this.getAlerts` callback for?

Reply:
ah! no need for this...will remove it